infra/gravity/commands.go

```diff
@@ -269,3 +282,17 @@ func (g *gravity) Reboot(ctx context.Context, graceful bool) error {
 	g.ssh = client
 	return nil
 }
+
+// PullLogs fetches essential logs from the host and stores them in state dir
+func (g *gravity) CollectLogs(ctx context.Context, prefix string) (string, error) {
+	if g.ssh == nil {
+		return "", trace.AccessDenied("node %v is poweroff", g)
+	}
+
+	files := []string{
+		"/var/lib/gravity/planet/log",
+		fmt.Sprintf("%s/*.log", g.installDir),
```

Comment:
TODO: we can pull everything from the log directory but I was talking about messages files written by the syslog.